pages/common/ssh-keygen.md

```diff
@@ -18,10 +18,6 @@
 
 `ssh-keygen -t rsa -b 4096 -C "{{email}}"`
```

Comment:
I think the token syntax is missing here
```suggestion
`ssh-keygen -t {{rsa}} -b {{4096}} -C "{{email}}"`
```

Reply:
i don't know about that , i'd say it only adds confusion **rsa** is an actual key type and not a place holder although it can be some other type as well, like **ecdsa** or **_dsa_** (not used anymore) same for the bit length 

as for the -C (comment part) it doesn't need to be an email, the option can be omitted entirely , than it will default to _current_user_name@current_host_name_ or can be any string like  _devkey_

Reply:
@aduzsardi the token syntax is not just for placeholders, but for anything which can be changed by the user whithout  changing what the command does.

> as for the -C (comment part) it doesn't need to be an email, the option can be omitted entirely , then it will default to _current_user_name@current_host_name_ or can be any string like _devkey_

I think we shouldn't completely remove `email` in this case, since it's used most of the time for this. How about
```suggestion
`ssh-keygen -t rsa -b 4096 -C "{{comment|email}}"`
```

Reply:
```suggestion
`ssh-keygen -t {{dsa|ecdsa|ed25519|rsa}}-b {{4096}} -C "{{email|email}}"`
```